docs/pages/reference/networking.mdx

```diff
@@ -124,45 +125,53 @@ numbers for each service.
-SSH Service or Kubernetes) are routed through the Proxy Service's public web address.
+TLS routing is enabled by default. In this mode, all connections to a Teleport
+service (e.g., the Teleport SSH Service or Kubernetes) are routed through the
+Proxy Service's public web address.
 
 Read more in our [TLS Routing](../architecture/tls-routing.mdx) guide.
 
-| Port | Service | Description |
+| Port | Downstream Service | Description |
 | - | - | - |
-| 443 | Proxy | In TLS Routing mode, the Proxy handles all protocols, including Web UI, HTTPS, Kubernetes, SSH, and all databases on a single port. |
-| 3021 | Proxy | Port used by Teleport Proxy Service instances to dial agents in Proxy Peering mode. |
-| 3022 | Node | SSH port. This is Teleport's equivalent of port `#22` for SSH. Only used when Teleport Node is replacing SSH.|
-| 3025 | Auth | TLS port used by the Auth Service to serve its API to other Nodes in a cluster.|
-| 3028 | Desktop | When using Desktop Service `windows_desktop_service.listen_addr` |
+| 443 | Proxy Service | In TLS Routing mode, the Proxy handles all protocols, including Web UI, HTTPS, Kubernetes, SSH, and all databases on a single port. |
+| 3021 | Proxy Service | Port used by Teleport Proxy Service instances to dial agents in Proxy Peering mode. |
+| 3022 | SSH Service | SSH port. This is Teleport's equivalent of port `#22` for SSH. Only used when Teleport agent is replacing SSH.|
+| 3028 | Desktop Service | When using Desktop Service `windows_desktop_service.listen_addr` |
 
-### Ports without TLS routing
+#### Ports without TLS routing
 
 In some cases, administrators may want to use separate ports for different services.
 In those cases, they can set up separate listeners in the config file.
 
-| Port | Service | Description |
+| Port | Downstream Service | Description |
 | - | - | - |
-| 3021 | Proxy | Port used by Teleport Proxy Service instances to dial agents in Proxy Peering mode. |
-| 3022 | Node | SSH port. This is Teleport's equivalent of port `#22` for SSH. |
-| 3023 | Proxy | SSH port clients connect to. The Proxy Service will forward this connection to port `#3022` on the destination Node. |
-| 3024 | Proxy | SSH port used to create "reverse SSH tunnels" from behind-firewall environments into a trusted proxy server. |
-| 3025 | Auth | TLS port used by the Auth Service to serve its API to other Nodes in a cluster. |
-| 3080 or 443 | Proxy | HTTPS connection to authenticate `tsh` users into the cluster. The same connection is used to serve a Web UI. |
-| 3026 | Kubernetes | HTTPS Kubernetes proxy `proxy_service.kube_listen_addr` |
-| 3027 | Kubernetes | Kubernetes Service `kubernetes_service.listen_addr` |
-| 3028 | Desktop | Desktop Service `windows_desktop_service.listen_addr` |
+| 3021 | Proxy Service | Port used by Teleport Proxy Service instances to dial agents in Proxy Peering mode. |
+| 3022 | SSH Service | SSH port. This is Teleport's equivalent of port `#22` for SSH. |
```

Comment:
Nit: Now that we've introduced a header for "Agent ports", a few of these ports can probably be removed from the "Proxy ports" section. A good example of this is `3022`, if someone stood up a Teleport instance that just had `proxy_service` enabled, then this port would not be opened.